tests/CMakeLists.txt

```diff
@@ -61,8 +66,8 @@ add_subdirectory(unittests)
 add_subdirectory(dumpmodel)
 
 # Tests that don't fit into one of the broad categories above
-CREATE_PODIO_TEST(ostream_operator.cpp "")
-CREATE_PODIO_TEST(write_ascii.cpp "")
+CREATE_PODIO_TEST(ostream_operator.cpp ROOT::Tree)
+CREATE_PODIO_TEST(write_ascii.cpp ROOT::Tree)
```

Comment:
Why do these now need to be explicitly linked against `ROOT::Tree`? In principle they are completely standalone and make no use of any ROOT features (at least they shouldn't).

Reply:
If linking to `ROOT::Tree` is necessary, we should make the datamodel library (target) depend on that already, I think.